index.js

```diff
@@ -72,15 +72,32 @@ LinkModule = (function() {
     tagXml = xmlTemplater.fileTypeConfig.tagsXmlArray[0];
     newText = this.getLinkXml({
       linkID : linkId,
-      linkText : text
+      linkText : text,
+      size: this.getLinkFontSize(xmlTemplater, templaterState.fullTextTag)
     });
     return this.replaceBy(newText, tagXml);
   };
 
+  LinkModule.prototype.getLinkFontSize = function(xmlTemplater, fullTextTag) {
```

Comment:
This function should find the previously defined font size. However, I have trouble determining the case when it is not defined at all (e.g. the link is part of a title `Blabla {^link}`). This works for me. If you have a better idea, let me know and I'll change it